Sources/UIDeviceExtensions.swift

```diff
@@ -85,6 +85,18 @@ extension UIDevice {
         return UIDevice.current.userInterfaceIdiom == UIUserInterfaceIdiom.pad
     }
 
+    /// EZSE: Returns path of documents directory
+    class func documentsDirectoryPath() -> String {
```

Comment:
Yea that sounds about right. I would wait for one of the collaborators to comment.